src/api/utils/summaries.py

```diff
@@ -1,17 +1,26 @@
 from ..models import (
+    Project,
     SummarySampleEventModel,
     SummarySampleEventSQLModel,
     SummarySiteModel,
     SummarySiteSQLModel,
 )
 
+project_statuses = [Project.OPEN, Project.LOCKED]
```

Comment:
Is this used in this file? Looks like only used in command, and this and the `Project` import could be removed here.